app/models/agents/twitter_user_agent.rb

```diff
@@ -64,6 +67,10 @@ def validate_options
       if options[:include_retweets].present? && !%w[true false].include?(options[:include_retweets])
         errors.add(:base, "include_retweets must be a boolean value string (true/false)")
       end
+      
+      if options[:exclude_replies].present? && !%w[true false].include?(options[:exclude_replies])
```

Comment:
Can you please explain this? I don't really know Ruby, sorry :)
It is `options[:exclude_replies].present?` `false` for existing agents?